docs/variants/tuxedo_ibs15/post_install.md

```diff
@@ -0,0 +1,24 @@
+# Post-installation setup
+
+This document contains extra steps to perform after installing Dasharo in order
+to enable full functionality.
+
+## Touchpad hotkey enablement (Linux)
+
+The touchpad hotkey needs extra setup to function correctly under Linux. To
+enable the touchpad hotkey to work under Linux, follow the steps below:
+
+1. Create a file `/etc/udev/hwdb.d/60-keyboard.hwdb` with the following contents:
+   ```
+   evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS50MU:*
```

Comment:
maybe we could contribute it?

Reply:
We certainly can. Such a change would go to the systemd repo like so: https://github.com/systemd/systemd/pull/17774

Reply:
Let's try to make it happen. @macpijan cc